fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalFilterOperator.java

```diff
@@ -22,6 +22,11 @@ public ScalarOperator getPredicate() {
         return predicate;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        return this == o;
```

Comment:
What's the difference with `==` ?

Reply:
None difference, the purpose is to check the object, may be same duplicate operators in memo,  will find wrong if use value eqauls check